catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.config.xml

```diff
@@ -148,6 +148,13 @@
             type="String"
             cardinality="10000"
             required="false"/>
+
+        <AD id="scheduleFrequencyList"
+            name="Query Schedule Frequency Lists"
+            description="Custom list of schedule frequencies in seconds.  This will override the frequency list in the query schedule tab.  Leave this empty to use the frequency list on the Catalog UI."
```

Comment:
These defaults are already hard coded in the Catalog UI; adding them here seemed redundant.